README.md

```diff
@@ -9,41 +9,65 @@
 - You can use it on the backend side
 - Token verification is part of [Apple sign-in](https://developer.apple.com/documentation/signinwithapplerestapi) process
 - The flow is
-    - Client app (iOS or Android) will redirect user to the OAuth2 login screen
-    - User will login
-    - App will receive the tokens
-    - App should send the `idToken` to the backend which will verify it
+  - Client app (iOS or Android) will redirect user to the OAuth2 login screen
+  - User will login
+  - App will receive the tokens
+  - App should send the `idToken` to the backend which will verify it
 - [Verification steps implemented](https://developer.apple.com/documentation/sign_in_with_apple/sign_in_with_apple_rest_api/verifying_a_user):
-    - Verify the JWS E256 signature using the server’s public key
-    - Verify the nonce for the authentication
-    - Verify that the iss field contains https://appleid.apple.com
-    - Verify that the aud field is the developer’s client_id
-    - Verify that the time is earlier than the exp value of the token
+  - Verify the JWS E256 signature using the server’s public key
+  - Verify the nonce for the authentication
+  - Verify that the iss field contains https://appleid.apple.com
+  - Verify that the aud field is the developer’s client_id
+  - Verify that the time is earlier than the exp value of the token
 
 ## Installation
+
 ```bash
 npm install verify-apple-id-token
 ```
 
 ## Usage
+
 ### Typescript
+
 ```typescript
-import verifyAppleToken from 'verify-apple-id-token';
+import verifyAppleToken from 'verify-apple-id-token'
 
 const jwtClaims = await verifyAppleToken({
-    idToken: 'yourIdToken',
-    clientId: 'yourAppleClientId',
-    nonce: 'nonce', // optional
-});
+  idToken: 'yourIdToken',
+  clientId: 'yourAppleClientId',
+  nonce: 'nonce', // optional
+})
+
+jwtClaims.email // get email of the user
 ```
 
 ### Javascript
+
 ```javascript
-const verifyAppleToken = require('verify-apple-id-token').default;
+const verifyAppleToken = require('verify-apple-id-token').default
 
 const jwtClaims = await verifyAppleToken({
-    idToken: 'yourIdToken',
-    clientId: 'yourAppleClientId',
-    nonce: 'nonce', // optional
-});
+  idToken: 'yourIdToken',
+  clientId: 'yourAppleClientId',
+  nonce: 'nonce', // optional
+})
 ```
+
+## Exports
```

Comment:
If you want to add methods and types to the readme add further documentation to be able to get to know what it is. For example describe in one or two sentences what is the `VerifyAppleIdTokenParams` and how to use it.

This is not so useful without description because everyone is able to look at the source code or inspect it in their IDE.

So you can add the description to the readme or remove it from readme and add comments to the source code on particular places.